### ISSUE TITLE
Allow drag & drop on the start page

### ISSUE THREAD

**Issue by borys-p** [RealFaviconGenerator/realfavicongenerator#380]
I had a file ready in Total Commander and tried to drop it on the web page opened in the backgroud... which didn't work, unfortunately. 

It would be nice if it did - a minor enhacement, albeit usefut.

**Comment by martin-braun** [comment]
Yes, this would be useful, I want this as well, please make it happen, it's really a basic thing and I think it should be implemented.
